pulsar-metadata/pom.xml

```diff
@@ -71,6 +71,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.pulsar</groupId>
```

Comment:
It doesn't look like the pulsar-metadata tests depend on this dependency.

Reply:
because i move the `TestZKServer`, and the `BaseMetadataStoreTest` in the pulsar-metadata module dependence on this class .

```java
public abstract class BaseMetadataStoreTest extends TestRetrySupport {
    protected TestZKServer zks;
    protected EtcdCluster etcdCluster;
```

https://github.com/apache/pulsar/blob/master/pulsar-metadata/src/test/java/org/apache/pulsar/metadata/BaseMetadataStoreTest.java#L39